src/screen/Council/CouncilScreen.tsx

```diff
@@ -69,5 +66,21 @@ export function CouncilScreen({navigation}: {navigation: NavigationProp<Dashboar
 }
 
 const styles = StyleSheet.create({
-  header: {justifyContent: 'space-between', flexDirection: 'row', padding: standardPadding * 2},
+  header: {
+    padding: standardPadding * 2,
+    flexDirection: 'row',
+    justifyContent: 'flex-end',
+  },
+  listHeader: {justifyContent: 'space-between', flexDirection: 'row', padding: standardPadding * 3},
+  content: {paddingVertical: standardPadding, paddingHorizontal: standardPadding * 2},
 });
+
+export function useCouncilMembers() {
```

Comment:
We are getting the same data in the `CouncilScreen` without caching it, can you extract this to its own file in the hook folder and reuse it?

Reply:
9c90f27 ✅